docs/JestObjectAPI.md

```diff
@@ -119,6 +119,92 @@ test('implementation created by jest.genMockFromModule', () => {
 });
 ```
 
+This is how `genMockFromModule` will mock the follwing data types:
+
+#### `Function`
+
+A new function will be created. The new function will have no formal parameters and when called will return `undefined`. This functionality also applies to `async functions`.
```

Comment:
Should this be "new mock function" and link to the mock function docs?

Reply:
most definitely, I'll make this update.